### PR TITLE
add header

### PR DESCRIPTION
**Pull request by VladGlivuk**

### CODE REVIEW

File: src/components/LeaderBoardHeader/LeaderBoardHeader.module.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.wrapper {
+  &__header {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 34px;
+    line-height: 32px;
+    color: #1a1c1d;
+    text-align: center;
```
Comment (ochornolutskyi) [inline src/components/LeaderBoardHeader/LeaderBoardHeader.module.scss:6]: Why you need & here?

File: src/components/LeaderBoardHeader/TopUser/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+interface ITopUserProps {
+  score: number;
+  name: string;
+}
+
+const TopUser: FC<ITopUserProps> = ({ score, name}) => {
+  return (
+    <div className={styles.main} >
+      <div className={styles.main__photo} >
+        <div className={styles.main__photo_score}>{score}</div>
```
Comment (ochornolutskyi) [inline src/components/LeaderBoardHeader/TopUser/index.tsx]: Is score a photo element?

File: src/components/LeaderBoardHeader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+import styles from "./LeaderBoardHeader.module.scss";
+
+const LeaderBoardHeader: FC = () => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.wrapper__header}>
+        Cube<span>19</span> Leaderboard
+      </div>
+      <div className={styles.wrapper__table}>
```
Comment (ochornolutskyi) [inline src/components/LeaderBoardHeader/index.tsx]: Should be classname table without wrapper

File: src/components/LeaderBoardHeader/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <span className={styles.wrapper__table_title}>
+          All time Highest Scorers
+        </span>
+        <div className={styles.wrapper__table_TopUsers}>
+          <TopUser
+            score={402}
```
Comment (ochornolutskyi) [inline src/components/LeaderBoardHeader/index.tsx:19]: If you pass only string, you shouldn't use a brackets. Only string as value

File: src/components/LeaderBoardHeader/LeaderBoardHeader.module.scss (hunk 1 above)
Comment (VladGlivuk) [reply src/components/LeaderBoardHeader/LeaderBoardHeader.module.scss:6]: > Why you need & here?

I need only font-size and text-align for title in my header

File: src/components/LeaderBoardHeader/index.tsx (hunk 3 above)
Comment (VladGlivuk) [reply src/components/LeaderBoardHeader/index.tsx]: > Should be classname table without wrapper

Everywhere?

**Pull request merged**